lib/msal-browser/src/error/NativeAuthError.ts

```diff
@@ -31,7 +31,15 @@ export const NativeAuthErrorMessage = {
     userSwitch: {
         code: "user_switch",
         desc: "User attempted to switch accounts in the native broker, which is not allowed. All new accounts must sign-in through the standard web flow first, please try again."
-    }
+    },
+    tokensNotFoundInCache: {
```

Comment:
We can use the existing no_tokens_found error and leverage logger messages if we need to know which code path led to it

Reply:
Modified